File: Day0/Solution.c
int main() {
    // Declare a variable named 'input_string' to hold our input.
    char input_string[105];

    // Read a full line of input from stdin and save it to our variable, input_string.
    scanf("%[^\n]", input_string);

    // Print a string literal saying "Hello, World." to stdout using printf.
    printf("Hello, World.\n");

    printf("%s", input_string);
    return 0;
}

File: Day0/Solution.go
package main

import (
    "bufio"
    "fmt"
    "os"
)
func main() {
    //bufio package is used to help reading from Stdin
    reader := bufio.NewReader(os.Stdin)
    //Read a line from Stdin into the 'input' variable
    input, _ := reader.ReadString('\n')

    fmt.Println("Hello, World.")
    fmt.Println(input)
}
